src/cr/cube/matrix.py

```diff
@@ -26,11 +26,33 @@ def __init__(self, unordered_matrix):
 
     @classmethod
     def matrix(cls, cube, dimensions, slice_idx):
-        """Return a TransformedMatrix object constructed from this cube result."""
+        """ -> TransformedMatrix object constructed from values of `cube`.
+
+        `cube` is the `cr.cube.Cube` object containing the data for this matrix. Note
+        that not all the data in `cube` will necessarily be used by this matrix. When
+        `cube` is more than 2-dimensional, it is "sliced" and each slice gets its own
+        matrix (and `_Slice` object).
+
+        `dimensions` is a pair (2-tuple) of (rows_dimension, columns_dimension)
+        Dimension objects. These are based on two of the dimensions of `cube` but may
+        not be the first two and may and often do have transformations applied that are
```

Comment:
They _are_ always the last two though, no? Maybe write it like that, it's more explicit. Unless I've forgotten something about how cube works 🤷